src/parser.rs

```diff
@@ -88,7 +118,7 @@ fn from_u8_array_to_i32(input: &[u8]) -> i32{
 }
 
 pub fn parse(input : &str) -> String {
-    let expression = operators(input.as_bytes());
+    let expression = parse_expression_top_level(input.as_bytes());
     println!("{:?}", expression);
     // FIXME: no explaination needed here unless you have Deco's IQ
```

Comment:
Insulting comment typo, what a shame for my guild

Reply:
I consciously avoided addressing this as I thought the humiliation you'd feel would be more powerful if you noticed this yourself